kivy/uix/bubble.py

```diff
@@ -235,6 +245,33 @@ def clear_widgets(self, **kwargs):
         else:
             content.clear_widgets()
 
+    def hide(self, *largs, **kwargs):
+        '''Close the Bubble if it is open.
+        When the view is dismissed, it will be faded out before being
+        removed from the parent. If you don't want animation, use::
+
+            view.dismiss(animation=False)
+
+        '''
+        parent = self.parent
+        if not parent:
+            return
+
+        if kwargs.get('animation', True):
+            anim = Animation(opacity=0, d=.225)
+            anim.bind(on_complete=lambda *args: parent.remove_widget(self))
+            anim.start(self)
+        else:
+            parent.remove_widget(self)
+
+    def on_touch_down(self, touch):
+        if not self.collide_point(*touch.pos):
+            if self.auto_dismiss:
+                self.hide()
+                return True
+        super(Bubble, self).on_touch_down(touch)
```

Comment:
I think you should return the result of the call to super, here, instead of always True.